makiflow/trainers/nn_render/render_trainer.py

```diff
@@ -124,10 +131,25 @@ def __init__(self, model_creation_function, exp_params: str, path_to_save: str):
                 Path to json file with parameters for training.
             path_to_save : str
                 Path for experiments folder. If its does not exist, it will be created.
+            restore_function : function
+                Function that restore image from network prediction of the neural render network.
+                By default restore function is:
+                    X - predict of the network
+                    Y - final image, then:
+                        Y = (X + 1) * 128
+                So, default normalization for images are next:
+                    I - image
+                    N - normalize result, then:
+                        N = I / 128 - 1
+            save_texture_info : bool
+                If true, in csv file will be written std, mean, kurtosis and asymmetry and stored in experiment folder.
```

Comment:
"If set to True, the  std, mean, kurtosis and asymmetry will be written into the csv file, which will be stored in the experiment folder."